server.js

```diff
@@ -23,7 +24,8 @@ var cors_proxy = require('./lib/cors-anywhere');
 cors_proxy.createServer({
   originBlacklist: originBlacklist,
   originWhitelist: originWhitelist,
-  requireHeader: ['origin', 'x-requested-with'],
+  targetHostWhitelist: targetHostWhitelist,
+  requireHeader: [],//['origin', 'x-requested-with'],
```

Comment:
Don't disable `requireHeader: ['origin', 'x-requested-with'],`. That's a security issue.